src/format/html/format-html.ts

```diff
@@ -75,11 +76,11 @@ export function htmlFormat(
     createHtmlFormat(figwidth, figheight),
     {
       metadataFilter: (metadata: Metadata) => {
-        if (!!metadata[kMinimal] && metadata[kTheme] === undefined) {
-          return {
-            ...metadata,
-            theme: "none",
-          };
+        if (metadata[kMinimal] === true) {
```

Comment:
deno linter throw me a comment if I use `!!metadata[kMinimal]` to be only `!metadata[kMinimal]`. 
I used explicit form to avoid confusion as in 
https://github.com/quarto-dev/quarto-cli/blob/217e24c7d50731cdcbeec493a5543b7a679be59b/src/format/html/format-html.ts#L395